bot/NanoTrasenBot.py

```diff
@@ -1056,7 +1056,6 @@ def autoRecv():
                      conn.privmsg(targetchannel,sender+" : "+C_sarcasticball.sarcasticball(influx.lower(),debug,truesender,nonhighlight_names,prefix)) 
             elif influx.lower()[0:len(Name)] == lowname and not influx.lower()[len(Name):].isalpha() or \

```

Comment:
If you're ditching the Yah??s then is the rest of the content under this elif of any use?